erpnext/accounts/party.py

```diff
@@ -74,15 +77,15 @@ def _get_party_details(party=None, account=None, party_type="Customer", company=
 		party_details["sales_team"] = [{
 			"sales_person": d.sales_person,
 			"allocated_percentage": d.allocated_percentage or None
-		} for d in ship_to.get("sales_team") or party.get("sales_team")]
+		} for d in party.get("sales_team")]
```

Comment:
Why are you removing ship_to functionality?

Reply:
Resolved with ed931e5